samples/python/greedy_causal_lm/lora.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import openvino_genai
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('models_path')
+    parser.add_argument('adapter_path')
+    parser.add_argument('prompt')
+    args = parser.parse_args()
+
+    device = 'CPU'  # GPU can be used as well
+    adapter = openvino_genai.Adapter(args.adapter_path)
+    adapter_config = openvino_genai.AdapterConfig()
+    adapter_config.add(adapter, 0.75)
+
+    pipe = openvino_genai.LLMPipeline(args.models_path, device, adapters=adapter_config)  # register all required adapters here
```

Comment:
In C++ version, we override alpha parameter in `generate` method, while Pipeline is created with default alpha value.

BTW, could you please also drop https://github.com/openvinotoolkit/openvino.genai/blob/master/samples/cpp/lora_greedy_causal_lm/ folder and move sample file to `greedy_causal_lm` as it's done for Python?